tools/opcode_metadata/generate_rs_opcode_metadata.py

```
@@ -0,0 +1,364 @@
+
+import tomllib
+from cpython import Analysis, get_analysis, get_stack_effect
+from opcodes import OpcodeInfo
+from utils import DEFAULT_INPUT, ROOT, get_conf, to_pascal_case
+
+OUT_FILE = ROOT / "crates/compiler-core/src/bytecode/opcode_metadata.rs"
+
+
+@dataclasses.dataclass(frozen=True, slots=True)
+class OpcodeGen:
+    info: OpcodeDef
```

Flake8 (7.3.0)

[error] undefined name 'OpcodeDef'

(F821)